### DISCUSSION TITLE
Unfortunately custom heads are not loaded after converting the world.

### DISCUSSION THREAD

**Discussion by xImNoxh** [Minestom/Minestom#958] [Help] [answered]
The Title say all

**Comment by KrystilizeNevaDies** [comment, marked as answer]
How are you converting the world? Have you registered a correct sign block handler?

**Reply by xImNoxh** [reply]
I made the world with World Converter. [there](https://github.com/Minestom/WorldConverter)

**Reply by TheMode** [reply]
Minestom supports Anvil, just put your map in the `world` folder in your project path.

I would suggest providing more information in the discussion message next time.
